inclusive-design/wecount.inclusivedesign.ca
# Fix: Image and text widget vertical alignment issue

## Description

Fixes the vertical alignment issue for image and text widget.

## Steps to test

1. Go to the admin panel
2. Create a new page using Image and text widget with image position `left`
3. Change the vertical alignment to different options
4. See the changes get applied correctly on the preview
